### ISSUE TITLE
Tray icon looks strange and doesn't show unread messages

### ISSUE THREAD

**Issue by DavidCdeB** [zulip/zulip-desktop#1224]
When I launch Zulip, the tray icon is shown in green (second from the left), looks strange and does not show unread messages:
![notif](https://user-images.githubusercontent.com/18029016/176286369-780bf0ed-d9e4-4e2f-9bec-74c9ebd1918d.png)

Notifications are enabled in Zulip's Settings

GNOME 42.2
Linux version 5.10

Many thanks for your help.

**Comment by Abastro** [comment]
Getting the same issues, I wonder if this is because it is installed through snap.
EDIT: Installed through apt, same issues. I inspected using dbus, the icon itself is rendered weirdly.

**Comment by DavidCdeB** [comment]
@Abastro  

> Getting the same issues, I wonder if this is because it is installed through snap.
EDIT: Installed through apt, same issues.

Thanks for reporting.
I use an Arch Linux-based distribution, hence these kind of programs are available in Arch Linux's AUR repository, where e.g. Zulip is to be found here: https://aur.archlinux.org/packages/zulip-desktop
I did not manually build, but used the usual automated tool: `yay -S zulip-desktop`

**Comment by Abastro** [comment]
Yep, in my case I installed it on ubuntu with zulip's official ppa. It seems universal on linux, at least those which rely on StatusNotifierItem protocol. (I use custom system tray)

**Comment by alxsimon** [comment]
Same here, dot icon without unread messages count.
Installed through Flatpak on Fedora 37.
(Zulip is the most-left tray icon)
![image](https://user-images.githubusercontent.com/9567176/204421469-34691007-efb4-4635-8394-315d11c17593.png)

**Comment by achton** [comment]
I also have this on Ubuntu 23.04, installed via the Zulip deb repository.

![image](https://github.com/zulip/zulip-desktop/assets/109079/650b5d69-ef65-4b5f-8951-17946f1a6f48)

**Comment by marnovandermaas** [comment]
Any solution for this yet? I have this issue with Zulip (v5.12.0) installed through snap on Ubuntu 24.04.

![Image](https://github.com/user-attachments/assets/d08f5b34-9e3a-46b0-99dc-5dfe450eb410)

**Comment by skandinaff** [comment]
I have the same issue on ubuntu 24.04, gnome 46, installed from snap and flatpak on two different machines - same broken icon.

**Comment by andypiper** [comment]
Came to report this issue on Fedora 43 / Wayland / GNOME 49; looks like it has been around for quite a while. Zulip 5.12.3 from Flathub.

It looks like various PRs claim to fix this and that it is something to do with fractional scaling (can confirm that I am using a fractional scaling factor).

e.g. related #1364 #1365 #1464 

Still broken where I am 🫤

**Comment by qay21** [comment]
This is still very much an issue, at least on Flatpak version 5.12.3.
